project/.travis/before_script_test.sh.twig

```diff
@@ -1,3 +1,12 @@
 #!/usr/bin/env bash
 set -ev
 
+{% if 'phpcr' in services %}
```

Comment:
better check for project name here: doctrine-phpcr-admin-bundle

Reply:
according to the mongo service example you gave me, this would be consistent.

Reply:
but having ci pipelines in mind, then services are additional software -  mostly containers, which is good for mongo, and here not :-1: